### ISSUE TITLE
Workspace - Fields appear twice in invite people page

### ISSUE THREAD

**Issue by kavimuru** [Expensify/Expensify.cash#4060]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:
1. Launch the app
2. Log in with expensifail account
3. Click on Green FAB
4. Select New Workplace
5. Choose the name and click Get Started
6. Click on People and click Invite
## Expected Result:
Fields appear once in the Invite people page.

## Actual Result:
Fields  appear twice in the Invite people page.

## Workaround:
Unknown

## Platform:
Where is this issue occurring?

 Web ✔️
 iOS ✔️
 Android ✔️
 Desktop App ✔️
 Mobile Web ✔️

**Version Number:**
1.0.78-0
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Notes/Photos/Videos:** 

**Expensify/Expensify Issue URL:**

https://user-images.githubusercontent.com/43996225/125713631-ad24b6ce-3824-4615-8264-7a19b9c30a82.mp4

<img src="https://user-images.githubusercontent.com/43996225/125713634-3be7e8b2-24b6-48d6-8435-262870ffdf9b.jpg" width="40%">

[View all open jobs on Upwork](https://www.upwork.com/ab/jobs/search/?q=Expensify%20React%20Native&sort=recency&user_location_match=2)

**Comment by MelvinBot** [comment]
Triggered auto assignment to @tgolen (`Engineering`), see https://stackoverflow.com/c/expensify/questions/4319 for more details.

**Comment by tgolen** [comment]
Hm, that's pretty weird, but I verified it happens! Must be rendered twice or something. Gonna try to look into this

**Comment by tgolen** [comment]
OK, I'm trying to debug this locally, but I am not able to create a workspace for some reason. I keep getting the error `Your session has expired. Please sign in again.` from the API and having trouble getting this working.

**Comment by tgolen** [comment]
All right, signing out of e.cash and signing back in seemed to have fixed it.

The problem is, I can't reproduce the problem locally:

![image](https://user-images.githubusercontent.com/1228807/125830049-17284f5d-0aaf-45d4-af2c-2df0d2784a27.png)

I'll try checking out staging to see if I can reproduce it with that code

**Comment by tgolen** [comment]
OK, so if I check out the staging branch locally, I can reproduce the bug.

However, if I apply the changes from https://github.com/Expensify/Expensify.cash/pull/3988 to the staging branch, then the bug is fixed.

What's odd is that the PR says it's on staging, but that doesn't seem possible.

**Comment by tgolen** [comment]
I'm working with @roryabraham to see if we can determine what went wrong. You can see in this PR https://github.com/Expensify/Expensify.cash/pull/4083 the difference between main and staging, when there should be no difference.

**Comment by tgolen** [comment]
Not really an update, but we are waiting to do a full deploy from `main` to `staging` to see if it fixes this.

**Comment by isagoico** [comment]
@tgolen We retested this on latest build (1.0.79-0) and it's still reproducible 😟

**Comment by tgolen** [comment]
@roryabraham Do you have any update for this?

**Comment by parasharrajat** [comment]
@tgolen I just checked the code for the page. It seems the merge Conflict was not resolved correctly. Both old and new changes are merged into the code which is causing duplicates. It could be me as well.

Lol. I think it's me. https://github.com/Expensify/Expensify.cash/blame/staging/src/pages/workspace/WorkspaceInvitePage.js

**Comment by roryabraham** [comment]
No update other than that [this diff](https://github.com/Expensify/Expensify.cash/compare/main...1.0.79-0) has the same weirdness we observed before. Unless other PRs have been merged since the lock label was added, there should be no diff between main and the latest tag.

**Comment by tgolen** [comment]
OK, I'm going to set this back to `daily` since we're just waiting on another deploy from `main` to `staging` to see if it clears up.

**Comment by roryabraham** [comment]
Well, there has been another deploy from `main` to `staging`, but for some reason it's still not registering the diff we expect. [This](https://github.com/Expensify/Expensify.cash/blob/96de1bc141d460d89d628f0bf5eccf187916820d/.github/workflows/updateProtectedBranch.yml#L57-L77) is the code that updates `staging` from `main`. If we look at [the logs for the latest run](https://github.com/Expensify/Expensify.cash/runs/3081837312?check_suite_focus=true), we can see that, when `main` is merged into the temporary copy of `staging`, `WorkspaceInvitePage` is not included in the [list of changed files](https://github.com/Expensify/Expensify.cash/runs/3081837312?check_suite_focus=true#step:9:15). The pull request that was created for this run is [right here](https://github.com/Expensify/Expensify.cash/pull/4103/files), and I would've expected it to include the changes in `WorkspaceInvitePage` from `main`.

Because I'm unsure why this happened and we've never seen it happen before, let's open a PR directly from `main` to `staging` and merge that. It [won't cause a staging deploy](https://github.com/Expensify/Expensify.cash/blob/d63fa31716663f491e85c69661a3806a6d5a05f0/.github/workflows/platformDeploy.yml#L18), so we'll have to unlock and relock the `StagingDeployCash` in order to trigger another staging deploy.

**Comment by parasharrajat** [comment]
Maybe we can modify the invite page and it will be included and there is an pr https://github.com/Expensify/Expensify.cash/pull/4068 here.

**Comment by roryabraham** [comment]
> Maybe we can modify the invite page and it will be included and there is an pr #4068 here.

Well, I can't explain why the existing diff from the `WorkspaceInvitePage` is not being included in the [PR that's supposed to update `main` from `staging`](https://github.com/Expensify/Expensify.cash/pull/4103/files), so I'm not sure that merging more changes to `main` will solve the problem. It might, but I don't know.

Instead, I think we should just merge https://github.com/Expensify/Expensify.cash/pull/4113 and then unlock and relock the `StagingDeployCash`.

**Comment by tgolen** [comment]
Not fixed! That PR merged staging into main... so main is broken again. I'll submit a PR to fix main and we can CP it
